sniffr/scripts/seed_db_activities.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy import create_engine
+from models import Activity
+import os
+from sqlalchemy.orm import sessionmaker
+
+def seed_db_activities():
```

Comment:
I could go either way.
Separate = Easier for each contributor to add their own section without conflicts.
Merged = Probably best long-term solution, but it may be big.